### PR TITLE
Fix broken code of conduct link on welcome page

### LINKED ISSUE #229: CNCF Code of Conduct link leads to a 404 page
The CNCF Code of Conduct link on this page https://www.kubernetes.dev/welcome/ leads to a 404 web page - https://www.kubernetes.dev/resources/code-of-conduct/ . Has it moved?

I see the code of the welcome page is here - https://github.com/kubernetes/contributor-site/blob/master/content/en/welcome/_index.md

Probably we need to link to this - https://www.kubernetes.dev/community/code-of-conduct/ ? or this - https://github.com/cncf/foundation/blob/master/code-of-conduct.md ? or this - https://kubernetes.io/community/static/cncf-code-of-conduct/ ?

or maybe link to this - https://github.com/kubernetes/contributor-site/blob/master/code-of-conduct.md ?

### PR DESCRIPTION
**Pull request by karuppiah7890**
Fixes #229

### CODE REVIEW

File: content/en/welcome/_index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ contributor resources, Kubernetes SIGs, and more.
 <h2>Get Involved:</h2>
 
 The first step is to check out the <a href="/docs/guide/">contributor guide</a>.
-All contributors must abide by the <a href="/resources/code-of-conduct/">CNCF Code of Conduct</a>
+All contributors must abide by the <a href="/community/code-of-conduct/">CNCF Code of Conduct</a>
```
Comment (idvoretskyi) [inline content/en/welcome/_index.md:36]: This link is broken. Better use - https://github.com/kubernetes/community/blob/master/code-of-conduct.md

**Review by idvoretskyi — CHANGES_REQUESTED** [review]

File: content/en/welcome/_index.md (hunk 1 above)
Comment (karuppiah7890) [reply content/en/welcome/_index.md:36]: https://www.kubernetes.dev/community/code-of-conduct/ in turn shows some content and also links to  https://git.k8s.io/community/code-of-conduct.md / https://github.com/kubernetes/community/blob/master/code-of-conduct.md

Comment (mrbobbytables) [reply content/en/welcome/_index.md:36]: @karuppiah7890 is correct here, we want to link to our version 👍 
/lgtm
/approve

**Pull request merged**